dev/CommonStyles/RepeatButton_themeresources.xaml

```diff
@@ -114,6 +117,11 @@
                                       HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
                                       VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
                                       AutomationProperties.AccessibilityView="Raw">
+
+                        <contract7Present:ContentPresenter.BackgroundTransition>
+                            <contract7Present:BrushTransition Duration="0:0:0.083" />
```

Comment:
>0:0:0.083 [](start = 72, length = 9)

Can we create a common resource for this value and share it ? Ideally we should be able to share the BrushTransition itself, but I think @teap hit some issues with sharing it.

Reply:
yup, will do:)

Reply:
Lets do this as a one shot and update everywhere in a separate PR once all the updates are done.